packs/data/feats.db/megavolt.json

```diff
@@ -11,15 +11,15 @@
             "value": 2
         },
         "description": {
-            "value": "<p>You bleed off some electric power from your innovation in the shape of a damaging bolt. Creatures in a @Template[type:line|distance:20] from your innovation take 3d4 electricity damage, with a basic Reflex save against your class DC. The electricity damage increases by 1d4 at 8th level and every 2 levels thereafter. [[/r floor(@actor.level/2)d4[electricity]]]{leveled electricity damage}</p>\n<p><strong>Unstable Function</strong> You overload and supercharge the voltage even higher. Add the unstable trait to Megavolt. The area increases to a @Template[type:line|distance:60] and the damage increases from d4s to d12s [[/r floor(@actor.level/2)d12[electricity]]]{leveled electricity damage (d12)}. If you have the breakthrough innovation class feature, you can choose a 60-foot or @Template[type:line|distance:90] for the area when you use an unstable Megavolt; if you also have the revolutionary innovation class feature, you can choose a 60-foot, 90-foot, or @Template[type:line|distance:120].</p>\n<p><strong>Special</strong> If your innovation is a minion, it can take this action rather than you.</p>"
+            "value": "<p>You bleed off some electric power from your innovation in the shape of a damaging bolt. Creatures in a @Template[type:line|distance:20] from your innovation take 3d4 electricity damage, with a @Check[type:reflex|dc:resolve(@actor.attributes.classDC.value)|basic:true] against your class DC. The electricity damage increases by 1d4 at 8th level and every 2 levels thereafter. [[/r floor(@actor.level/2)d4[electricity]]]{leveled electricity damage}</p>\n<p><strong>Unstable Function</strong> You overload and supercharge the voltage even higher. Add the unstable trait to Megavolt. The area increases to a @Template[type:line|distance:60] and the damage increases from d4s to d12s [[/r floor(@actor.level/2)d12[electricity]]]{leveled electricity damage (d12)}. If you have the breakthrough innovation class feature, you can choose a 60-foot or @Template[type:line|distance:90] for the area when you use an unstable Megavolt; if you also have the revolutionary innovation class feature, you can choose a 60-foot, 90-foot, or @Template[type:line|distance:120].</p>\n<p>@Check[type:flat|dc:17|name:Unstable Flat Check|showDC:all|traits:unstable]{Unstable Check}</p>\n<p><strong>Special</strong> If your innovation is a minion, it can take this action rather than you.</p>"
```

Comment:
```suggestion
            "value": "<p>You bleed off some electric power from your innovation in the shape of a damaging bolt. Creatures in a @Template[type:line|distance:20] from your innovation take 3d4 electricity damage, with a @Check[type:reflex|dc:resolve(@actor.attributes.classDC.value)|basic:true] against your class DC. The electricity damage increases by 1d4 at 8th level and every 2 levels thereafter. [[/r floor(@actor.level/2)d4[electricity]]]{leveled electricity damage}</p>\n<p><strong>Unstable Function</strong> You overload and supercharge the voltage even higher. Add the unstable trait to Megavolt. The area increases to a @Template[type:line|distance:60] and the damage increases from d4s to d12s [[/r floor(@actor.level/2)d12[electricity]]]{leveled electricity damage (d12)}. If you have the breakthrough innovation class feature, you can choose a 60-foot or @Template[type:line|distance:90] for the area when you use an unstable Megavolt; if you also have the revolutionary innovation class feature, you can choose a 60-foot, 90-foot, or @Template[type:line|distance:120].</p>\n<p>@Check[type:flat|dc:17|name:Unstable Flat Check]{Unstable Check}</p>\n<p><strong>Special</strong> If your innovation is a minion, it can take this action rather than you.</p>"
```